actors/evm/src/interpreter/instructions/memory.rs

```diff
@@ -215,4 +215,296 @@ mod tests {
         assert_eq!(mem.len(), 32);
         assert_eq!(&mem[0..4], result_data);
     }
+
+    #[test]
+    fn test_mload_nothing() {
+        evm_unit_test! {
+            (rt, m) {
+                PUSH0;
+                MLOAD;
+            }
+
+            let result = m.step();
+            assert!(result.is_ok(), "execution step failed");
+            let result = m.step();
+            assert!(result.is_ok(), "execution step failed");
```

Comment:
We should probably start writing these lines as `m.step().expect` (or even just use unwrap, they're easy enough to debug).